packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -55,6 +58,14 @@ export interface DescriptionListProps extends Omit<React.HTMLProps<HTMLDListElem
     xl?: string;
     '2xl'?: string;
   };
+  termAutoFitMinModifier?: {
```

Comment:
All these props need a prop description to clarify their purpose. The new prop is setting the width of the term column in horizontal mode at various breakpoints. The previous prop - `autoFitMinModifier` - also needs a prop description.

Also, the default value should likely replace the need for the new `termWidth` prop, and i think the name of your new `termAutoFitMinModifier` prop is not quite correct. I propose possibly `horizontalTermWidths`